src/components/containers/ContainersListContent.js

```diff
@@ -12,28 +12,36 @@ import {list, listTemplateActions} from "../../modules/containers/actions";
 import {actionsToButtonList} from "../../utils/templateActions";
 
 const TABLE_COLUMNS = [
+  {
+    title: "Name",
+    dataIndex: "name",
+  },
   {
     title: <><BarcodeOutlined style={{marginRight: "8px"}} /> Barcode</>,
     dataIndex: "barcode",
     render: (barcode, container) => <Link to={`/containers/${container.id}`}>{barcode}</Link>,
   },
   {
-    title: "Name",
-    dataIndex: "name",
+    title: "Sample",
+    dataIndex: "samples",
+    render: (samples, container) =>
+      (container.kind == "tube" && samples.length > 0) ? 
+        <Link to={`/samples/${samples[0].id}`}>{samples[0].name}</Link> : 
+        null,
   },
   {
     title: "Kind",
     dataIndex: "kind",
   },
   {
-    title: "Children",
-    dataIndex: "children",
-    align: 'right',
-    render: children => children ? children.length : null,
+    title: "Location Name",
+    dataIndex: "location",
+    render: location => location ? <>{location.name}</> : null,
```

Comment:
`location.name`, not `<>{location.name}</>`

Reply:
yeah this is part of the work in progress patch i did to get rid of some errors ... Must have forgot to save when i tested location.name...